cleanlab_studio/internal/clean_helpers.py

```diff
@@ -52,4 +52,13 @@ def poll_cleanset_status(
 
         if res["has_error"]:
             pbar.set_postfix_str(res["step_description"])
+            error_message = res["error_message"]
+            error_type = res["error_type"]
+
+            if error_type is not None:
+                assert (
+                    error_message is not None
+                ), "Error message should be present for handled error"
```

Comment:
if `error_message` isn't present for some reason, we wouldn't want to show this opaque assertion error to a user. we'd instead want to show them a default error message and send this telemetry back to the backend